be/src/exec/short_circuit_hybrid.cpp

```diff
@@ -113,6 +115,8 @@ Status ShortCircuitHybridScanNode::get_next(RuntimeState* state, ChunkPtr* chunk
         *eos = true;
     }
     *chunk = std::move(result_chunk);
+    _num_rows_returned += (*chunk)->num_rows();
+    COUNTER_SET(_rows_returned_counter, _num_rows_returned);
```

Comment:
still use the runtime_profile in ScanNode or ExecNode?  
so only _rows_returned_counter has non-zero values, other counters have zero values?
The zero values is filtered in final profile result?

Time spent on seeking pk in pk index  is not taken into consideration?

Reply:
right now scan node only register basic counter in its profile, other counter won't register because scannode::prepare won't be called, more profile counter can be added later, this pr's goal is making short-circuit point query has a basic profile to distinguish from normal query.